### PR TITLE
Updated the CI1, CI2 , & CI3 tasks to refer to 2017.

### PR DESCRIPTION
**Pull request by namantw**
Inside the "_projects" folder :
use_coala.md, use_coala_2.md and use_coala_3.md :

- Changed all occurrences of 2016 to 2017.
- Changed the project links with the new ones.

Closes https://github.com/coala/projects/issues/433

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: _projects/use_coala.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ can be used to help select a repository that meets those criteria.
 To filter out repositories that do not meet the first two requirements, combine
 
 1. [`stars:>99`](https://github.com/search?o=desc&q=stars%3A%3E%3D99&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93), and
-2. [`pushed:>2016-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2016-09-30&type=Repositories&ref=searchresults)
+2. [`pushed:>2017-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2016-09-30&type=Repositories&ref=searchresults)
```
Comment (nalinbhardwaj) [inline _projects/use_coala.md]: change URL, still points to old one.

File: _projects/use_coala.md [hunk 2]
```diff
@@ -74,9 +74,9 @@ can be used to help select a repository that meets those criteria.
...
 
-The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2016-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
+The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2017-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
 has more than 28,000 repository results, so there are many to choose from.
 
 An optional additional search term is
```
Comment (nalinbhardwaj) [inline _projects/use_coala.md]: Change the URL for this one too to new version

File: _projects/use_coala.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
----
+--
```
Comment (nalinbhardwaj) [inline _projects/use_coala.md]: undo this

Comment (namantw) [reply _projects/use_coala.md]: I am working on it. Please wait, I'll ping you when I am done. 😄

**Comment by namantw** [comment]
@nalinbhardwaj Looks like I created a separate pull request. Please close this pull request and review https://github.com/coala/projects/pull/435

**Pull request closed without merging**